### PR TITLE
Merge develop, add migrations, hooks and tests

### PR DESCRIPTION
**Pull request by codiebeulaine**

### CODE REVIEW

**Comment by codiebeulaine** [comment]
@saeedmb8 @justinvdm ready for review

**Comment by miltontony** [comment]
👍

**Comment by miltontony** [comment]
hold up..

**Comment by justinvdm** [comment]
@codie it is likely we will need this same thing for other things that will be edited via the CMS (blogs and faqs are the only other things at the moment, I think?).

Happy for us to do that as a separate PR though.

File: mentorship/mentor/tests/test_views.py [hunk 1]
```diff
@@ -442,14 +442,22 @@ def test_category_list_all(self):
         category.save_revision().publish()
         category2 = Category(title='Test Category2', slug='test-category2')
         self.main.add_child(instance=category2)
-        category2.save_revision().publish()
+        category2.save_revision()
+        category2.unpublish()
+        category3 = Category(title='Test Category3', slug='test-category3')
+        self.main.add_child(instance=category3)
+        category3.save_revision().publish
```
Comment (justinvdm) [inline mentorship/mentor/tests/test_views.py]: Should this be `publish()`?

**Comment by justinvdm** [comment]
Left some comments, looks good :)

**Comment by KaitCrawford** [comment]
:+1:

**Pull request merged**